AAI/AAIConnectProfile.md

```diff
@@ -581,12 +624,18 @@ Payload:
 
 -   `addtional claims`: OPTIONAL. Any other additional non-GA4GH claims are allowed. This specification does not dictate the format of other claims.
 
-#### Claims sent to Data Holder by a Broker via /userinfo
+#### Claims sent to Data Holder by a Broker via `/userinfo`
 
 Only the GA4GH claims truly must be as prescribed here. Refer to OIDC Spec for
 more information. The /userinfo endpoint MAY use `application/json` or
-`application/jwt`. If `application/jwt` is returned, it MUST be signed as per
-[UserInfo](https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse).
+`application/jwt`.  It is RECOMMENDED that if desiring to return a JWT, a token endpoint supporting AAI token exchange exists to do that and `/userinfo` returns a general blob.
+
+If `application/jwt` is returned, it MUST be signed as per
+[UserInfo](https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse).  
```

Comment:
can we clarify tighten this to say that if JWTs are being returned it is expected that it is *only* in JWS Compact Serialization? Or if we want to support both Compact and JSON then explicitly mention that.
https://datatracker.ietf.org/doc/html/rfc7515#section-3.1  and 3.2
The linked section of the openid spec is silent on the issue - and whilst I think it is obvious that for HTTP headers people would be using the compact serialisation, where the value is returned as a HTTP response I am not sure it is so clear that the JSON serialisation is wrong.

Reply:
Agree with both of you trying to improve the text here. We should clarify that this is not designed to be a bearer token. If returning a JWT from UserInfo it is for proof that it came from the IdP and not hijacked man in the middle. TLS alone is good enough for most folks, but someone who wants added assurances can optionally return signed JWTs. It should explicitly state in JWS Compact Serialization. For bearer tokens, yes, please use a token endpoint.

Reply:
@andrewpatto - JWT is only defined for JWS or JWE compact serialization, so did you want to specify JWS vs. JWE?
I believe that /userinfo should be (and should have been) only JSON. In particular, it serves multiple scopes that are not pertinent to passports; itself being a scope in its own right.